system/modules/playos-status.nix

```diff
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+let
+  ttyNumber = 8;
+  tty = "tty${toString ttyNumber}";
+  ttyPath = "/dev/${tty}";
+  driverUrl = "http://127.0.0.1:8382";
+
+  script =
+    pkgs.writeShellScriptBin "print-status" ''
+      while :; do
+        screen=$(xrandr --current | grep '*' | awk '{print $1}')
+        networkCount=$(connmanctl services | grep wifi | wc -l)
+        rfid=$(opensc-tool --list-readers)
+        controller=$(systemctl is-active playos-controller)
+        time=$(date +'%T %Z')
+        printf "\033c"
+        printf "%s\n" \
```

Comment:
Since we don't display a whole lot of data at the moment, `\n\n` for empty dividing lines could be nice for legibility.

Reply:
:+1: